app/views/teacher_mailer/references_reminder.text.erb

```diff
@@ -0,0 +1,19 @@
+<% @referees_to_be_reminded = @application_form.work_histories.select{ |history| history.reference_request.should_display_teacher_reference_reminder_text?}.map{|work_history| work_history.reference_request} %>
+Dear <%= application_form_full_name(@application_form) %>
+
+We’re still waiting for a response from one or more of the references you provided to verify your work history.
+
+<% if @referees_to_be_reminded.first.reminder_emails.count == 1 %>
+  We contacted your references on <%= @application_form.work_histories.first.reference_request.requested_at.to_date.to_fs(:long_ordinal_uk) %> but have not received a response from:
+<% elsif @referees_to_be_reminded.first.reminder_emails.count == 2 %>
+  The following references have just 2 weeks to respond to the reference request.
+<% end %>
+
+<% @referees_to_be_reminded.each do |reference_request|  %>
+    <%= reference_request.contact_name + ". " + reference_request.work_history.school_name %>
+<% end %>
+
+
+They need to respond by <%= @application_form.work_histories.first.reference_request.expires_at.to_date.to_fs(:long_ordinal_uk) %>. If your references do not respond, and, as a result, we cannot verify your work history, we may not be able to award you QTS. You should contact your reference to remind them about the request.
```

Comment:
`@application_form.work_histories.first.reference_request` means we'll always show the value for the first work history, even if we're not sending an email about the first work history.